### ISSUE TITLE
anonymous functions missing from stacktrace in Phonegap/Cordova on iOS

### ISSUE THREAD

**Issue by drbild** [MindscapeHQ/raygun4js#79]
Anonymous functions are excluded from the stacktrace in the `UIWebView` on iOS 7.  This affects all Phonegap / Cordova apps running on iOS.

The regexs used by tracekit (see `computeStackTraceFromStackProp()`) to extract the stack information don't appear to be designed for this version of WebKit. The regex for Gecko ends up being used, but fails to extract anonymous functions.

Gecko format: `<function_name>@<file_url>:<line_number>:<column_number>`

iOS 7 `UIWebView` format (named function): `<function_name>@<file_url>:<line_number>:<column_number>`
iOS 7 `UIWebView` format (anon function): `<file_url>:<line_number>:<column_number>`

Basically, Gecko always includes the `@` - even for anonymous functions - while Webkit does not.  Simply making the `@` optional in the regex fixes the issue, but it seems wrong to rely on the Gecko regex in the first place.
